public/ACMW/css/index.css

```diff
@@ -125,10 +125,19 @@
 /*.dropdown-menu li a:hover {
 	background-color: red !important;
 } */
+footer .container{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)